tests/test10.cpp

```diff
@@ -72,7 +72,6 @@ int main (int argc, char **argv) {
 
   app = gtk_application_new ("org.gtk.example", G_APPLICATION_FLAGS_NONE);
   // this next line is necessary to ensure that the gtkmm internals are initialized
-  Gtk::Main::init_gtkmm_internals ();
```

Comment:
This line is still necessary, as otherwise you get a lot of warnings and the canvas cannot be interacted with. They changed the name of the function though, so you will need to apply the following patch (I cannot push to your branch 😢  ):

```diff
commit 7e2ada536920099ef0b92b7850b206abb119c7de
Author: Tom Schoonjans <Tom.Schoonjans@genomicsplc.com>
Date:   Sun Oct 1 13:19:16 2023 +0100

    init internals in test10

diff --git a/tests/test10.cpp b/tests/test10.cpp
index 78ac5b0..603347f 100644
--- a/tests/test10.cpp
+++ b/tests/test10.cpp
@@ -18,7 +18,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 /* This examples demonstrates how to use Gtkmm-PLplot from Gtk */
 
 #include "gtkmm-plplot.h"
-#include <gtkmm.h>
+#include <gtkmm/init.h>
 
 #ifndef M_PI
 #define M_PI (3.14159265358979323846)
@@ -72,6 +72,7 @@ int main (int argc, char **argv) {
 
   app = gtk_application_new ("org.gtk.example", G_APPLICATION_FLAGS_NONE);
   // this next line is necessary to ensure that the gtkmm internals are initialized
+  Gtk::init_gtkmm_internals();
   g_signal_connect (app, "activate", G_CALLBACK (activate), NULL);
   status = g_application_run (G_APPLICATION (app), argc, argv);
   g_object_unref (app);

```